packages/@n8n/nodes-langchain/nodes/vendors/OpenAi/methods/listSearch.ts

```diff
@@ -109,9 +131,9 @@ export async function assistantSearch(
 		};
 	} else {
 		return {
-			results: (data || []).map((assistant: IDataObject) => ({
-				name: assistant.name as string,
-				value: assistant.id as string,
+			results: (data || []).map((assistant) => ({
+				name: assistant.name ?? '',
```

Comment:
maybe we should use description or id instead of empty string here

Reply:
Makes sense, updated